Sources/SwiftDocC/Model/Rendering/Tutorial/References/DownloadReference.swift

```diff
@@ -28,7 +28,12 @@ public struct DownloadReference: RenderReference, URLReference {
     public var url: URL
     
     /// The SHA512 hash value for the resource.
-    public var sha512Checksum: String?
+    public var checksum: String?
+
+    @available(*, deprecated, renamed: "checksum")
+    public var sha512Checksum: String {
```

Comment:
This is still API breaking because there's no setting defined here. Should we implement one to preserve compatibility?